packages/core/src/App/Containers/Layout/header/traders-hub-header.tsx

```diff
@@ -0,0 +1,190 @@
+        payload.filter(config => {
+            if (config.link_to === routes.mt5) {
+                return !is_logged_in || is_mt5_allowed;
+            }
+            return true;
+        });
+
+    const toggleModal = () => {
+        if (!has_any_real_account) {
+            toggleReadyToDepositModal();
+        } else if (window.location.pathname === routes.traders_hub) {
+            toggleNeedRealAccountForCashierModal();
+        }
+    };
+
+    const handleClickCashier = () => {
+        if ((!has_any_real_account && is_virtual) || real_account_needed_for_cashier) {
+            toggleModal();
+        } else {
+            history.push(routes.cashier_deposit);
+        }
+    };
+
+    const AccountInfoComponent = React.useCallback(
+        () => (
+            <AccountInfo
+                acc_switcher_disabled_message={account_switcher_disabled_message}
+                account_type={account_type}
+                balance={formatMoney(currency, balance ?? '', true)}
+                country_standpoint={country_standpoint}
+                currency={currency}
+                is_dialog_on={is_accounts_switcher_on}
+                is_disabled={false}
+                is_eu={is_eu}
+                is_virtual={is_virtual}
+                toggleDialog={toggleAccountsDialog}
+            />
+        ),
+        [
+            account_switcher_disabled_message,
+            account_type,
+            balance,
+            country_standpoint,
+            currency,
+            is_accounts_switcher_on,
+            is_eu,
+            is_virtual,
+            toggleAccountsDialog,
+        ]
+    );
+
+    return (
+        <header
+            className={classNames('traders-hub-header', {
+                'traders-hub-header--is-disabled': is_app_disabled || is_route_modal_on,
+                'traders-hub-header--is-hidden': platforms[platform as keyof TPlatforms],
+            })}
+        >
+            <div className='traders-hub-header__menu-left'>
+                <MobileWrapper>
+                    <ToggleMenuDrawer {...{ platform_config: filterPlatformsForClients(platform_config) }} />
+                    {header_extension && is_logged_in && <div>{header_extension}</div>}
+                </MobileWrapper>
+                <div
+                    className={classNames('traders-hub-header__logo-wrapper', {
+                        'traders-hub-header__logo-wrapper--cashier': cashier_routes,
+                    })}
+                >
+                    <StaticUrl href='/'>
+                        <DerivBrandLogo className='traders-hub-header__logo' />
+                    </StaticUrl>
+                </div>
+                <DesktopWrapper>
+                    <div className='traders-hub-header__divider' />
+                    <TradersHubHomeButton />
+                </DesktopWrapper>
+                <MenuLinks {...{ is_traders_hub_routes: true }} />
```

Comment:
```suggestion
                <MenuLinks is_traders_hub_routes />
```

Reply:
since MenuLinks is still in jsx, ts throws this problem. could there be any beautiful way of resolving this?
![Screenshot 2023-09-01 at 12 34 37 PM](https://github.com/binary-com/deriv-app/assets/122449658/bb5c09db-fcbb-4f35-8d1c-6532b2271b04)

Reply:
Guess we can't do anything about this ts error rather than spreading over the props for now. 
Thanks for the clarification 👍